distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -499,12 +499,12 @@ type Vector a
     map self function =
         Vector.new self.length i-> function (self.at i)
 
-    ## Applies a function to each element of the vector, returning the vector
+    ## Applies a function to each element of the vector, returning the `Vector`
```

Comment:
Sometimes we don't want to replace the `Vector` word in the `Array` documentation. I am then writing the `Vector` with capital first letter and quoting it.